src/sources/dnstap/mod.rs

```diff
@@ -72,6 +72,10 @@ pub struct DnstapConfig {
     ///
     /// This should not typically needed to be changed.
     pub socket_send_buffer_size: Option<usize>,
+
+    /// The namespace to use for logs. This overrides the global settings
```

Comment:
```suggestion
    /// The namespace to use for logs. This overrides the global settings.
```

Pretty sure the place we've all copied this from didn't have punctuation 😄